app/views/layouts/__header.html.erb

```diff
@@ -0,0 +1,37 @@
+
+<header class="sticky-top">
+  <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+    <a class="navbar-brand headTextSize" href="/"><%= image_tag"pepe_hacker.gif", class:"headerPic"%>DripBox</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav mr-auto w-100 justify-content-end">
+
+        <!-- Links können in Kombination mit dem mittleren Suchfeature realisiert werden -->
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="/data" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><!-- //////////////////////link hier für Daten-->
+            Data
+          </a>
+          <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+            <a class="dropdown-item" href="#">Dokumente</a> <!-- Link für eigene Dokumentseite -->
+            <a class="dropdown-item" href="#">Bilder</a> <!-- Link für eigene Bilderseite -->
+            <div class="dropdown-divider"></div>
+            <a class="dropdown-item" href="#">Für mich freigegeben</a> <!-- Link für Seite, die alle für mich freigegebenen Dateien  -->
```

Comment:
Bitte `link_to` Viewhelper bentuzen, statt alles in HTML selbst schreiben. Das hat den Vorteil, dass Rails darauf Einfluss nehmen kann, wo es muss, und damit weniger Aufwand bei späteren Features und Änderungen entsteht.